infrastructure/main.tf

```diff
@@ -34,4 +37,68 @@ module "db-notif" {
   subscription       = var.subscription
 }
 
+module "notification-scheduler-db-flexible" {
+  providers = {
+    azurerm.postgres_network = azurerm.postgres_network
+  }
+  source = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+  env    = var.env
+  product       = var.product
+  component     = var.component
+  business_area = "CFT" # sds or cft
+  # The original subnet is full, this is required to use the new subnet for new databases
+  subnet_suffix = "expanded"
+  pgsql_databases = [
+    {
+      name : var.database_name
+    }
+  ]
+  pgsql_version = "15"
+  # The ID of the principal to be granted admin access to the database server.
+  # On Jenkins it will be injected for you automatically as jenkins_AAD_objectId.
+  # Otherwise change the below:
+  admin_user_object_id = var.jenkins_AAD_objectId
+  common_tags = var.common_tags
+  pgsql_server_configuration = [
+    {
+      name  = "azure.extensions"
+      value = ""//need to get value for this by running "SELECT * FROM pg_extension;"
```

Comment:
do we need to update this after DB is created ?

Reply:
no, this needs to replicate the extensions that exist on the current db

Reply:
I've updated this now with the extensions copied from the existing db